src/sentry/utils/event_frames.py

```diff
@@ -0,0 +1,19 @@
+from typing import Any, MutableMapping
+
+
+def supplement_filename(platform: str, data_frames: MutableMapping[str, Any]):
```

Comment:
Can we add tests around this? As we add in more platforms it'll become more important. Can be in a follow up if you prefer.

Reply:
Yeah, definitely wanna add tests when we add more platforms. Will do some followup PRs for the new platforms when I get to it.